airbyte-integrations/bases/connector-acceptance-test/pyproject.toml

```diff
@@ -12,7 +12,7 @@ homepage = "https://github.com/airbytehq/airbyte"
 
 [tool.poetry.dependencies]
 python = "^3.10"
-airbyte-protocol-models = "*"
+airbyte-protocol-models = ">=0.4.0,<0.5.0"
```

Comment:
Should we upper bound to a major even? I guess there would be no breaking changes until a major is changed which mean minor changes are probably fine